pcs_detection/scripts/data/training_config.json

```diff
@@ -0,0 +1,69 @@
+{
+    "MODE": "TRAIN",
+    "WEIGHT_ID": "new_repo_test",
+    "WEIGHT_DIR": "data/weights",
```

Comment:
I would think this directory is relative to the location of the config? That doesn't look correct in this case. /scripts/data/data/weights isn't correct.

Reply:
training config and script moved to scripts directory